client/a8c-for-agencies/sections/purchases/licenses/license-details/actions.tsx

```diff
@@ -113,6 +119,19 @@ export default function LicenseDetailsActions( {
 				</Button>
 			) }
 
+			{ ( isPressableLicense || isWPCOMHostingLicense ) &&
+				licenseState !== LicenseState.Revoked &&
+				! isClientLicense && (
+					<Button
+						compact
+						href={
+							isPressableLicense ? '/marketplace/hosting/pressable' : '/marketplace/hosting/wpcom'
```

Comment:
We have `A4A_MARKETPLACE_HOSTING_WPCOM_LINK`  and `A4A_MARKETPLACE_HOSTING_PRESSABLE_LINK` constants we can use for these links. This way we only have one place to update when the URL scheme changes.

Reply:
Thanks, I'll update it!